modules/microshift-install-rhel-image-mode-cni.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-run-image-cni_{context}"]
+= Checking the {microshift-short} bootc image for the CNI
```

Comment:
```suggestion
= Configuring the {microshift-short} bootc image for the CNI
```
The checking part on the way is not the essence of the operation, so we might not want to put it in the title.